client/src/Visualizations/LineChart/LineChart.jsx

```diff
@@ -217,26 +247,96 @@ class LineGraph extends React.Component {
 
     return (
       <svg width={this.props.width} height={this.props.height}>
+        <FillBottom
+          xScale={xScale}
+          yScale={yDataScale}
+          {...this.props}
+          data={this.props.data}
+          color="#edf3ff"
+        />
         <Lines
           xScale={xScale}
           yScale={yDataScale}
           {...this.props}
           data={this.props.data}
-          color="#cbcfd6"
+          color="#ccddff"
+        />
+        {/*<DataCircles*/}
+        {/*  xScale={xScale}*/}
+        {/*  yScale={yDataScale}*/}
+        {/*  {...this.props}*/}
+        {/*  data={this.props.data}*/}
+        {/*  color="#edf3ff"*/}
+        {/*/>*/}
+        <FillBottom
+          xScale={xScale}
+          yScale={yDataScale}
+          {...this.props}
+          data={this.props.measure3}
+          color="#ddeaff"
```

Comment:
Why `{...this.props}` in the middle? And where does `xScale` and `yDataScale` come from ?

Reply:
Honestly I'm not entirely too sure what {...this.props} does, I just have it there because I copied the code from the <Lines/> and <DataCircles/>. 

Also, to my understanding, the yDataScale is the scale for the KCals (vs being the life expectancy).